e2e_test/streaming/encrypt.slt

```diff
@@ -0,0 +1,26 @@
+query T
```

Comment:
Seems only part of https://github.com/postgres/postgres/blob/master/contrib/pgcrypto/expected/rijndael.out

Shall we:
* Acknowledge they are from pgcrypto tests
* Explain the reason why some of the tests are not imported / supported.

Reply:
A more complete solution: #14896